app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable, :confirmable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Good job adding `:confirmable` to your User model authentication. However, as required for the for the project, you should ensure that User cannot be able to login before his `email` has been confirmed, because it is not the same in your case. Right now, after a user has singed up, he is immediately logged in even if without being confirmed. Make sure you fix that.
![Screenshot from 2023-08-03 10-45-10](https://github.com/prius2055/rails-blogging-app/assets/83042742/e0783b13-5fc2-4e50-806f-60ecd2114ed4)

Reply:
I used a separate gem letter opener as stmp failed to send a confirm mail